frontend/app/package.json

```diff
@@ -186,41 +96,32 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "fetch-mock": "^9.11.0",
     "hard-source-webpack-plugin": "^0.13.1",
     "jest": "^27.4.3",
     "jest-canvas-mock": "^2.4.0",
     "jest-fetch-mock": "^3.0.3",
     "jest-github-actions-reporter": "^1.0.3",
     "jest-websocket-mock": "^2.4.0",
-    "marked": "^4.2.12",
-    "message-channel": "^2.0.0",
-    "mock-socket": "^9.1.5",
     "prettier": "^2.8.3",
     "prop-types": "^15.7.2",
-    "react-responsive-carousel": "^3.2.23",
     "react-scripts": "5.0.1",
     "source-map-explorer": "^2.5.3",
     "start-server-and-test": "^1.15.3",
-    "timezone-mock": "^1.3.6",
     "typescript": "^4.9.5",
     "util": "^0.12.5",
     "webpack": "5.76.0"
   },
   "resolutions": {
     "@types/html-minifier-terser": "^7.0.0",
```

Comment:
I'm not exactly sure which ones of these I can remove but I removed the ones that existed in the dependencies and got removed such as `d3`.